src/index.js

```diff
@@ -131,6 +131,7 @@ class AlertsPlugin {
     } else {
       alarm.Properties.ExtendedStatistic = definition.statistic
     }
+    alarm.Properties = Object.assign(alarm.Properties, (definition.cloudFormation || {}));
     return alarm;
```

Comment:
This looks like it will overwrite the entire `Properties` object which works for your `customAlarmWithCloudformationEscapeHatch` example. However, what if I want to change just one property of a built-in alarm? Does it support that?
```yaml
definitions:
  functionDuration:
    cloudFormation:
      EvaluateLowSampleCountPercentile: ignore
```
Could you use something like the following? I'm not a javascript dev so maybe there is a better way...
```javascript
_.merge(alarm.Properties, (definition.cloudFormation || {}));
return alarm
```

Reply:
`Object.assign` does a shallow merge of properties, so existing properties keys will be preserved, only overriden keys in the `cloudFormation` object will overwrite the existing `properties` keys.